### PR TITLE
feat: adds faq page

### LINKED ISSUE #104: Page - FAQ

### PR DESCRIPTION
**Pull request by twomeaningsof**
Closes #104

### CODE REVIEW

File: src/components/Accordion/AccordionItem.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <div className="w-[358px] select-none xl:w-[552px]">
+      <div
+        className={twMerge(
+          'flex cursor-pointer items-center justify-between rounded-[4px] border-2 border-transparent bg-white py-4 pl-6 pr-8 drop-shadow-[8px_8px_33px_rgba(0,0,0,0.07)] transition-all duration-[100ms]',
+          isActive && 'border-quaternary'
+        )}
+        onClick={e => {
+          setIsActive(prev => !prev);
+          !isActive && e.currentTarget.scrollIntoView({ block: 'center' });
```
Comment (michal-karalus) [inline src/components/Accordion/AccordionItem.tsx:21]: It could be moved to separate const

File: src/pages/faq.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  KtoMozeDolaczyc,
+  JakDolaczyc,
+  WJakichObszarach
+];
+
+const itemsMarkdownPart2 = [
+  JakieSaKorzysci,
+  CzegoSieSpodziewac,
+  JakieSaObowiazki
+];
```
Comment (michal-karalus) [inline src/pages/faq.tsx:21]: Could it be merged into one array?

Comment (twomeaningsof) [reply src/pages/faq.tsx:21]: I couldn't find any other solution that would fully work. If it is merged into one, and I use solution proposed by Jędrzej it results in something like that:
image

File: src/components/Accordion/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { AccordionItem } from './AccordionItem';
+
+type AccordionProps = {
+  items: string[];
+};
+
+export const Accordion = ({ items }: AccordionProps) => (
+  <div className="flex flex-col gap-6">
```
[the remark this answers is not in the document]
Comment (twomeaningsof) [reply src/components/Accordion/Accordion.tsx:8]: If it is merged into one, and I use this solution, it results in something like that:
image

**Pull request merged**